packages/postcss-merge-rules/src/lib/ensureCompatibility.js

```diff
@@ -11,31 +11,15 @@ const cssFirstLine = 'css-first-line';
 const cssInOutOfRange = 'css-in-out-of-range';
 const formValidation = 'form-validation';
 
-/** @type {string[]} */
-const prefixes = [
-  '-ah-',
-  '-apple-',
-  '-atsc-',
-  '-epub-',
-  '-hp-',
-  '-khtml-',
-  '-moz-',
-  '-ms-',
-  '-o-',
-  '-rim-',
-  '-ro-',
-  '-tc-',
-  '-wap-',
-  '-webkit-',
-  '-xv-',
-];
+const vendorPrefix =
+  /-(ah|apple|atsc|epub|hp|khtml|moz|ms|o|rim|ro|tc|wap|webkit|xv)-/;
```

Comment:
I mean `/-(.+)-/` to avoid adding new vendor (but I think we won't have new ones)

Reply:
OK, I've updated the regex. Though, in the future there also could be a string that looks like a vendor prefix but has a  different meaning :-)